Server/VikingVault.Services/UnqiueEmailService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using VikingVault.DataAccess;
+using VikingVault.Services.Abstractions;
+
+namespace VikingVault.Services
+{
+    public class UniqueEmailService : IUniqueEmailService
+    {
+        private readonly VikingVaultDbContext _dbContext;
+
+        public UniqueEmailService(VikingVaultDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+
+        public bool? IsUniqueEmail(string email)
+        {
+
+            try
+            {
+                var ReturnedEmail = _dbContext.User.SingleOrDefault(s => s.Email == email);
```

Comment:
This variable name does not comply with the C# naming conventions and also is not suggestive enough. Please find a more suggestive name.